tests/unit/components/DonationSummary.spec.ts

```diff
@@ -46,147 +51,105 @@ describe( 'DonationSummary.vue', () => {
-				},
-			},
-		} );
-		expect( $t ).toBeCalledWith( 'donation_confirmation_topbox_donor_type_company' );
-		const params = findTranslationCallParams( 'language_item', $t.mock.calls );
-		const expectedFields = [ 'fullName', 'streetAddress', 'postalCode', 'city' ];
-		expectedFields.map( ( fieldName: string ) => expect( params.address ).toContain( address[ fieldName ] ) );
-	} );
+	const emptyAddress: Address = {
+		salutation: '',
+		title: '',
+		firstName: '',
+		lastName: '',
+		fullName: '',
+		companyName: '',
+		street: '',
+		postcode: '',
+		city: '',
+		country: '',
+		email: '',
+	};
 
-	function getLanguageItemMock(): ( key: string, values?: Record<string, string> ) => string {
-		return ( key: string, values: Record<string, string> = {} ) => {
-			if ( key !== 'language_item' ) {
-				return key;
-			}
-			let translated = '{interval} {formattedAmount} {paymentType} {personType} {address} {email}';
-			for ( const [ recordKey, value ] of Object.entries( values ) ) {
-				translated = translated.replace( `{${ recordKey }}`, `${ recordKey }:${ value }` );
-			}
-			return translated;
-		};
+	interface Props {
+		address: Address;
+		payment: { interval: any; amount: number; paymentType: any };
+		bankData: { iban: string; bic: string; bankName: string };
+		countries: Country[];
+		salutations: Salutation[];
 	}
 
-	const getWrapper = ( address ): VueWrapper<any> => {
-		const $t = getLanguageItemMock();
-		const $n = jest.fn( x => x );
+	const mocks = {
+		$t: ( key: string ) => key,
+	};
 
+	function mountWrapper( propsData: Props ): VueWrapper<any> {
 		return mount( DonationSummary, {
-			props: {
-				payment,
-				address: address,
-				addressType: 'person',
-				countries,
-				salutations,
-				languageItem: 'language_item',
-			},
+			props: propsData,
 			global: {
-				mocks: {
-					$t,
-					$n,
-				},
+				mocks,
 			},
 		} );
-	};
+	}
 
-	it( 'translates payment information', () => {
-		const wrapper = getWrapper( {} );
+	it( 'renders payment headline, amount, interval, and paymentType', () => {
+		const wrapper = mountWrapper( {
+			address: emptyAddress,
+			payment,
+			bankData: emptyBankData,
+			countries,
+			salutations,
+		} );
+		const text = wrapper.text();
 
-		expect( wrapper.find( '.form-summary-content' ).text() ).toStrictEqual( [
-			'interval:{"key":"donation_form_payment_interval_12"}',
-			'formattedAmount:{"amount":14.99,"key":"currency","currencyDisplay":"name"}',
-			'paymentType:{"key":"BEZ"}',
-			'personType:donation_confirmation_topbox_donor_type_person',
-			'address:donation_confirmation_review_address_missing',
-			'email:donation_confirmation_review_email_missing',
-		].join( ' ' ) );
+		expect( text ).toContain( 'donation_form_payment_headline' );
+		expect( text ).toContain( '{"amount":14.99,"key":"currency","currencyDisplay":"name"}' );
+		expect( text ).toContain( '{"key":"donation_form_payment_interval_12"}' );
+		expect( text ).toContain( 'donation_summary_via {"key":"BEZ"}' );
```

Comment:
You need a another test case with a donation where the payment type is not set, to test the other translation key